sdks/python/hatchet_sdk/runnables/types.py

```diff
@@ -82,31 +80,6 @@ class WorkflowConfig(BaseModel):
 
     task_defaults: TaskDefaults = TaskDefaults()
 
-    def _raise_for_invalid_expression(self, expr: str) -> None:
-        if not expr.startswith("input."):
-            return None
-
-        _, field = expr.split(".", maxsplit=2)
-
-        if field not in self.input_validator.model_fields.keys():
-            raise ValueError(
-                f"The concurrency expression provided relies on the `{field}` field, which was not present in `{self.input_validator.__name__}`."
```

Comment:
this is broken, since you could have a key like `foo.bar + foo.baz` or `foo.bar.baz`